spec/models/user_deletion_spec.rb

```diff
@@ -379,54 +379,59 @@ def association_key(association)
   end
 
   describe "WHEN the user has created a changeset" do
-    let(:repository) { FactoryGirl.create(:repository) }
-    let(:associated_instance) { FactoryGirl.build(:changeset, :repository_id => repository.id,
-                                                          :committer => user.login) }
+    with_created_filesystem_repository do
+      let(:associated_instance) do
+        FactoryGirl.build(:changeset,
+                          repository_id: repository.id,
+                          committer: user.login)
+      end
 
-    let(:associated_class) { Changeset }
-    let(:associations) { [:user] }
 
-    before do
-      Setting.enabled_scm = Setting.enabled_scm << "Filesystem"
-    end
+      let(:associated_class) { Changeset }
+      let(:associations) { [:user] }
 
-    it_should_behave_like "created journalized associated object"
+      it_should_behave_like "created journalized associated object"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.